tuiexporter/internal/tui/component/trace.go

```diff
@@ -119,15 +120,17 @@ func getTraceInfoTree(spans []*telemetry.SpanData) *tview.TreeView {
 	return tree
 }
 
-func appendAttrsSorted(parent *tview.TreeNode, attrs map[string]any) {
-	keys := make([]string, 0, len(attrs))
-	for k := range attrs {
+func appendAttrsSorted(parent *tview.TreeNode, attrs pcommon.Map) {
+	keys := make([]string, 0, attrs.Len())
+	attrs.Range(func(k string, _ pcommon.Value) bool {
 		keys = append(keys, k)
-	}
+		return true
+	})
 	sort.Strings(keys)
 
 	for _, k := range keys {
-		attr := tview.NewTreeNode(fmt.Sprintf("%s: %s", k, attrs[k]))
+		v, _ := attrs.Get(k)
+		attr := tview.NewTreeNode(fmt.Sprintf("%s: %s", k, v.AsString()))
```

Comment:
In this case, I think it's enough to just use [`AsString()`](https://pkg.go.dev/go.opentelemetry.io/collector/pdata/pcommon#Value.AsString)